docker-compose-dev.yml

```diff
@@ -42,7 +42,7 @@ services:
 
   web:
     restart: "unless-stopped"
-    image: cernopendata/static # Use this to make sure that COD3 Python-code image is built only once.
+    image: registry.cern.ch/cernopendata/cernopendata-portal:latest
```

Comment:
Thanks for checking. Let's check a different approach then. None of the comments above are valid anymore with this approach

Reply:
I've tested the static image asset gathering use case described above, and the update fixes it. :+1: 

However I'm still seeing some issues, e.g. a build problem related to overwriting assets:

```console
$ docker compose -f docker-compose-dev.yml build
...
9.858 Traceback (most recent call last):
9.858   File "/opt/invenio/var/instance/python/bin/cernopendata", line 33, in <module>
9.858 Collect static from blueprints
9.858     sys.exit(load_entry_point('cernopendata', 'console_scripts', 'cernopendata')())
9.858   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/click/core.py", line 1157, in __call__
9.859     return self.main(*args, **kwargs)
9.859   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/click/core.py", line 1078, in main
9.859     rv = self.invoke(ctx)
9.859   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/click/core.py", line 1688, in invoke
9.859     return _process_result(sub_ctx.command.invoke(sub_ctx))
9.859   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/click/core.py", line 1434, in invoke
9.859     return ctx.invoke(self.callback, **ctx.params)
9.859   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/click/core.py", line 783, in invoke
9.859     return __callback(*args, **kwargs)
9.859   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/click/decorators.py", line 33, in new_func
9.860     return f(get_current_context(), *args, **kwargs)
9.860   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/flask/cli.py", line 357, in decorator
9.860     return __ctx.invoke(f, *args, **kwargs)
9.860   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/click/core.py", line 783, in invoke
9.860     return __callback(*args, **kwargs)
9.860   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/invenio_assets/cli.py", line 24, in collect
9.860     current_assets.collect.collect(verbose=verbose)
9.860   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/flask_collect/collect.py", line 50, in collect
9.860     return storage.run()
9.860   File "/opt/invenio/var/instance/python/lib/python3.9/site-packages/flask_collect/storage/link.py", line 49, in run
9.860     os.symlink(f, destination)
9.860 FileExistsError: [Errno 17] File exists: '/opt/invenio/var/instance/python/lib/python3.9/site-packages/invenio_previewe
r/static/css/pdfjs/viewer.css' -> '/opt/invenio/var/instance/static/css/pdfjs/viewer.css'
9.876 /usr/lib64/python3.9/site-packages/XRootD/client/finalize.py:46: DeprecationWarning: Importing 'itsdangerous.json' is d
eprecated and will be removed in ItsDangerous 2.1. Use Python's 'json' module instead.
9.876   if isinstance(obj, File) and obj.is_open():
9.876
------
failed to solve: process "/bin/sh -c scripts/create-instance.sh" did not complete successfully: exit code: 1
```

Reply:
Thanks for pointing this out.  I have the feeling that master has the same issue since [this commit](https://github.com/cernopendata/opendata.cern.ch/commit/81d791e3d36115d0a8dd857c6420b1307eff2c01#diff-840af82fcc44e800f4f1ab340166bac7564e39b3a2e5b822a6781fdd3d69d32cR46). Could you please confirm that the patch in scripts/create_instance fixes it for you as well?